swagger.yaml

```
@@ -44,21 +422,255 @@ paths:
+    description: "The requested resource does not exist."
+    schema:
+      $ref: '#/definitions/ErrorList'
+  PreconditionFailed:
+    description: |
+      Change rejected due to condition not being met. This is typically because the entity-tag in the `If-Match` request header does not match the current entity-tag of the resource.
+    schema:
+      $ref: '#/definitions/ErrorList'
+  UnauthorisedError:
+    description: Authentication information is missing or invalid
+    schema:
+      $ref: "#/definitions/ErrorList"
 definitions:
-  HelloResponse:
+  Bundles:
+    description: "A list of bundles"
+    type: object
+    allOf:
+      - $ref: "#/definitions/PaginationFields"
+      - type: object
+        properties:
+          items:
+            type: array
+            items:
+              $ref: "#/definitions/Bundle"
+  Bundle:
+    description: "A model for the response body when retrieving a bundle"
+    allOf:
+      - type: object
+        properties:
+          bundle_type:
+            description: "The type of bundle."
+            type: string
+            enum:
+              - manual
+              - scheduled
+            example: "scheduled"
+          contents:
+            type: array
+            readOnly: true
+            items:
+              $ref: "#/definitions/ContentItem"
+          creator:
+            description: "The user that created the bundle, taken from the auth token supplied.  This should only be set on initial creation of the bundle."
+            type: string
+            example: "publisher@ons.gov.uk"
+          created_date:
+            description: "The time and date the bundle was created on."
+            type: string
+            format: date-time
+            readOnly: true
+            example: "2025-04-04T07:00:00.000Z"
+          id:
+            type: string
+            description: "The bundle id"
+            readOnly: true
+            example: "9e4e3628-fc85-48cd-80ad-e005d9d283ff"
+          last_updated_by:
+            description: "The user that last updated the bundle, taken from the auth token supplied."
+            type: string
+            example: "publisher@ons.gov.uk"
+          preview_teams:
+            description: "A list of teams who have permissions to view the dataset series in the bundle.  Supplied by AWS cognito."
+            type: array
+            items:
+              type: string
+          publish_date_time:
+            description: "The time and date of publication."
+            type: string
+            format: date-time
+            example: "2025-04-04T07:00:00.000Z"
+          state:
+            description: "The approval status of the overall bundle (TBC - values provided are examples only)."
+            type: string
+            enum:
+              - not_started
+              - in_progress
+              - approved
+            example: "approved"
+          title:
+            type: string
+            description: "The title of the bundle"
```

This is a writable, required field that can not be empty so should include:

```suggestion
            description: "The title of the bundle"
            minLength: 1
```

---

This has been added